api/types/types.proto~

```diff
@@ -0,0 +1,2043 @@
+syntax = "proto3";
```

Comment:
Looks like this file was accidentally checked in.

Reply:
:scream: